language/use-cases/document-summarization/summarization_with_documentai.ipynb

```diff
@@ -384,12 +385,10 @@
    "source": [
     "import glob\n",
     "import os\n",
-    "import textwrap\n",
     "from typing import Dict, List\n",
     "\n",
     "# If you already have a Document AI Processor in your project, assign the full processor resource name here.\n",
     "processor_name = processor.name\n",
-    "chunk_size = 5000\n",
```

Comment:
I think the explanatory text above on line 371 about `we are taking 5000 as a chunk_value` is outdated now?

Reply:
Good catch!